tests/icon_cache.py

```diff
@@ -1,41 +1,39 @@
 #!/usr/bin/env python3
+
 """
-# Copyright (C) 2016
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License (version 3+) as
-# published by the Free Software Foundation. You should have received
-# a copy of the GNU General Public License along with this program.
-# If not, see <http://www.gnu.org/licenses/>.
+Copyright (C) 2019 Numix Project
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License (version 3+) as
+published by the Free Software Foundation. You should have received
+a copy of the GNU General Public License along with this program.
+If not, see <http://www.gnu.org/licenses/>.
 """
+
+from json import load
 from os import path
-import json
-from collections import OrderedDict
 
-from utils import error
+from utils import error, space_check
+
 
 ABS_PATH = path.dirname(path.abspath(__file__))
 DB_FILE = path.join(ABS_PATH, "../data.json")
 
 with open(DB_FILE, 'r') as db_obj:
-    data = json.load(db_obj, object_pairs_hook=OrderedDict)
```

Comment:
This test loaded the JSON file as an ordered dictionary which isn't needed here.

Reply:
Are you sure about this? if you load the json file here, it will result in an unordered dict.

Reply:
The point was more that the `icon_cache` test doesn't need to load the JSON in a special way, the standard load used elsewhere is fine. I didn't test whether `object_pairs_hook=OrderedDict` ordered the dictionary, I just assumed it did based on the name.

Reply:
sorry I misunderstood what you said then :p it's perfect! 
the object_pairs_hook uses a new object for each child on your json object, as dict don't preserve the order, using OrderDict makes it use the same order as the one on the file

Reply:
Ahhhh, I get you. That actually makes a lot of sense, thanks! Also if anything else about this pull catches your eye as iffy let me know, I realise it's changing a lot of code you originally wrote

Reply:
You can ping me here whenever you need that. Will take a closer look once you think it should be merged ^^